src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -0,0 +1,154 @@
+import sortBy from 'lodash/sortBy';
+import property from 'lodash/property';
+import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
+import styled from '@emotion/styled';
+
+import {isUrl, percent} from 'app/utils';
+import {t} from 'app/locale';
+import AsyncComponent from 'app/components/asyncComponent';
+import UserAvatar from 'app/components/avatar/userAvatar';
+import DeviceName from 'app/components/deviceName';
+import ExternalLink from 'app/components/links/externalLink';
+import GlobalSelectionLink from 'app/components/globalSelectionLink';
+import Pagination from 'app/components/pagination';
+import TimeSince from 'app/components/timeSince';
+import space from 'app/styles/space';
+import {Group, Tag, TagValue} from 'app/types';
+
+type RouteParams = {
+  groupId: number;
```

Comment:
@leeandher So `groupId` comes out as an integer and not a string? Interesting.

Reply:
You're totally right, my bad -- its a string